src/test/java/pl/zimi/zadanka/Zad7Test.java

```diff
@@ -0,0 +1,18 @@
+package pl.zimi.zadanka;
+
+import org.junit.jupiter.api.Test;
+
+/*
+    Source: https://leetcode.com/problems/maximum-depth-of-binary-tree/
+ */
+
+public class Zad7Test {
+    @Test
+    void leetexampleTest() {
+        /*
+        Input: root = [3,9,20,null,null,15,7]
```

Comment:
wydaje mi się że najlepiej jakby jako API była metoda która przyjmuje TreeNode
czyli nie specjalnie przejmujemy się jaki jest input
zakładamy że mamy gotowe zbudowane drzewo i metoda przyjmuje to drzewo
wcześniej natomiast robimy kawałek kodu który czyta dane z wejścia i buduje drzewo

odpowiedziałem? czy niespecjalnie?

Reply:
poprawione zrobione metoda wywołujaca przetwarzanie danych w drzewo i wywołujaca szukanie maxdepth jest w klasie testów